lib/util/git/index.ts

```diff
@@ -636,21 +639,49 @@ export async function isBranchModified(branchName: string): Promise<boolean> {
     return isModified;
   }
 
-  logger.debug('branch.isModified(): using git to calculate');
-
   await syncGit();
-  // Retrieve the author of the most recent commit
-  let lastAuthor: string | undefined;
+  // Retrieve the commit authors
+  let branchAuthors: string[] = [];
   try {
-    lastAuthor = (
-      await git.raw([
-        'log',
-        '-1',
-        '--pretty=format:%ae',
-        `origin/${branchName}`,
-        '--',
-      ])
-    ).trim();
+    if (baseBranch) {
+      logger.debug(
+        `branch.isModified(): using git to calculate authors between ${branchName} and ${baseBranch}`
+      );
+      branchAuthors = [
+        ...new Set(
+          (
+            await git.raw([
+              'log',
+              '--pretty=format:%ae',
+              `origin/${branchName}..origin/${baseBranch}`,
+              '--',
+            ])
+          )
+            .trim()
+            .split('\n')
+        ),
+      ];
+      logger.debug(`Git authors in branch: ${branchAuthors.join(', ')}`);
```

Comment:
I was planning to log this only if multiple were found

Reply:
Do you have ideas on how to test that flow? I wasn't able to get it to work yesterday and figured having more logging could be helpful for debugging

Reply:
Doesn't it just happen if there's multiple git authors in the branch, after it was branched off the base branch?

Reply:
It does but wasn't able to get code coverage working for that case since doing a `git push` in jest resulted in things blowing up since the clone isn't mocked, from what i can tell

Reply:
OK, keep it as-is then